internal/planner/commit.go

```diff
@@ -238,7 +242,8 @@ func (n *dagScanNode) Next() (bool, error) {
 		n.queuedCids = append(make([]*cid.Cid, len(heads)), n.queuedCids...)
 
 		for i, h := range heads {
-			n.queuedCids[len(heads)-i-1] = &h.Cid
+			link := h // TODO remove when Go 1.22
```

Comment:
todo: Please link this todo to https://github.com/sourcenetwork/defradb/issues/2431

Reply:
✅